### PR TITLE
Add basic skeleton of unikernel runtime

### PR DESCRIPTION
**Pull request by Crazykev**
relay on #178 , ref #181

### CODE REVIEW

**Comment by feiskyer** [comment]
@Crazykev Seems this PR should make a rebase, some changes are already in.

**Comment by Crazykev** [comment]
@feiskyer Thanks for reminding, updated!

File: pkg/manager/manager.go [hunk 1]
```diff
@@ -70,6 +74,8 @@ func NewFraktiManager(
 	streamingServer streaming.Server,
 	privilegedRuntimeService runtime.RuntimeService,
 	privilegedImageService runtime.ImageService,
+	unikernelRuntimeService runtime.RuntimeService,
+	unikernelImageService runtime.ImageService,
```
Comment (feiskyer) [inline pkg/manager/manager.go]: should make those services params more general? e.g. wrap them under `runtime.RuntimeService` and change params to `(hyperService RuntimeService, streamingServer , priviledgedService RuntimeService, extraRuntimes [] RuntimeService)`

Comment (feiskyer) [inline pkg/manager/manager.go]: And pass in unikernelService inside array `extraRuntimes`.

Comment (Crazykev) [inline pkg/manager/manager.go]: em...That looks better if we add more runtime. How can we decide which is unikernelRuntime or other extra runtimes and assign to corresponding field in `FraktiManager`?

Comment (feiskyer) [inline pkg/manager/manager.go]: Yep, that's a problem. Let's keep it as is now.

**Pull request merged**